### DISCUSSION TITLE
头像问题

### DISCUSSION THREAD

**Discussion by ACZSHOP** [halo-dev/halo#5023] [General]
以前的版本还可以直接从附件里面选择一张图片当作用户头像，现在更新到2.11.2又没有这个功能了。只能重新上传，而且不能直接使用全图必须要经过截取部分内容
![2 0](https://github.com/halo-dev/halo/assets/142337549/6366a1a3-08f6-4b6b-9a4f-400a5064e457)
![2 1](https://github.com/halo-dev/halo/assets/142337549/1b49a50c-311b-412f-856d-ec3505c1f8c4)

**Comment by LIlGG** [comment]
可以使用滑轮缩小图片。
